pkgs/build-support/fetchzip/default.nix

```
@@ -45,7 +45,6 @@
     '' else ''
       mv "$unpackDir" "$out"
     '')
-    + extraPostFetch
     # Remove write permissions for files unpacked with write bits set
```

```suggestion
    + ''
    ${extraPostFetch}
    ''
    # Remove write permissions for files unpacked with write bits set
```

I would prefer this.

---

Changed.